### DISCUSSION TITLE
WalletConnectConnector

### DISCUSSION THREAD

**Discussion by summer233333** [wevm/wagmi#2613] [Q&A]
![image](https://github.com/wagmi-dev/wagmi/assets/40909307/8ca2d6e6-eefd-46d6-a534-831357d10e5b)

When configuring this place, three chains were set, but when the app scanned and connected, only one chain was received for configuration

**Comment by jxom** [comment]
What do you mean?

**Comment by summer233333** [comment]
![image](https://github.com/wagmi-dev/wagmi/assets/40909307/f4aabdd4-b035-4d4f-bff8-ff0b72fb9e4e)
Shouldn't the above configuration display three chains here

**Reply by jxom** [reply]
Looks like that Wallet doesn't fully support WalletConnect v2. You will need to get in touch with them to implement Optional Namespaces...
